docs/settingUp.md

```diff
@@ -88,9 +88,14 @@ Eclipse IDE is our preferred development environment. Support requests related t
       ![devverification-guide-1.png](images/devverification-guide-1.png)
      
    * Indentation: In TEAMMATES, we use 4 spaces in place of tabs for indentations.
-     Go to `Eclipse → Preferences` and configure for all the languages used in TEAMMATES:
-     * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`
-     * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`
+   * Trailing whitespace: all forms of trailing whitespace is disallowed, blank lines included.
+   * Go to `Eclipse → Preferences` and configure for all the languages used in TEAMMATES:
+     * Java:
+        * `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`
+        * `Java → Editor → Save Actions → Additional actions → Configure → Remove trailing whitespace → All lines`
```

Comment:
I feel it falls under the same category of using spaces instead of tabs. If the tab policy instructions are provided, why not the trailing spaces instructions?

The other 3/4 actions, however, are up to the developer. I wouldn't want to even mention them in the instructions.